base/workflow/R/run.write.configs.R

```diff
@@ -22,9 +22,10 @@
 #' @export
 #'
 #' @author David LeBauer, Shawn Serbin, Ryan Kelly, Mike Dietze
-run.write.configs <- function(settings, write = TRUE, ens.sample.method = "uniform", 
+run.write.configs <- function(settings, write = TRUE,  
                               posterior.files = rep(NA, length(settings$pfts)), 
-                              overwrite = TRUE) {
+                              overwrite = TRUE,
+                              input_design) {
```

Comment:
a couple things, one small, one big

small: it's generally best practice in R to put required parameters before those with defaults

BIG: is this a required parameter? or do we want to retain the capability for run.write.configs to generate designs? If input_design = NULL then what design to you generate? My gut says to make this required, and that the design of run.write.configs will ultimately be a lot simpler and cleaner for the separation of design generation from config writing.